### DISCUSSION TITLE
recalibration routine in DIANN

### DISCUSSION THREAD

**Discussion by Apirog9** [vdemichev/DiaNN#291] [Q&A]
Hello,

How data are recalibrated in DIANN during analysis? What kind of function is applied, is it performed in RT dependent mode, or one recalibration function is used on whole run?

Artur

**Comment by vdemichev** [comment]
Hi Artur,

I am not sure I understood the question. In general, the algorithms of DIA-NN are detailed in the Nature Methods paper.

Best,
Vadim

**Comment by Apirog9** [comment]
Sorry, it was somehow stuck in my mind that there is a mass correction step. Sometimes it would be beneficial, if data are not collected in a perfect way. Is it possible for DIA-NN to report MS1 mass errors for at least part of the detected precursors? It would enable recalibration of mzml file. It may be highly beneficial.

**Reply by vdemichev** [reply]
DIA-NN does do mass calibration, yes. Reporting mass errors is currently not possible, but might be implemented in the future.
